File: features/payment/payment_setting/src/main/java/com/tokopedia/payment/setting/add/model/CcIframe.java

package com.tokopedia.payment.setting.add.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CcIframe {

    @SerializedName("merchant_code")
    @Expose
    private String merchantCode;
    @SerializedName("profile_code")
    @Expose
    private String profileCode;
    @SerializedName("ip_address")
    @Expose
    private String ipAddress;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("user_id")
    @Expose
    private String userId;
    @SerializedName("customer_name")
    @Expose
    private String customerName;
    @SerializedName("customer_email")
    @Expose
    private String customerEmail;
    @SerializedName("callback_url")
    @Expose
    private String callbackUrl;
    @SerializedName("cc_token")
    @Expose
    private String ccToken;
    @SerializedName("signature")
    @Expose
    private String signature;

    public String getMerchantCode() {
        return merchantCode;
    }

    public void setMerchantCode(String merchantCode) {
        this.merchantCode = merchantCode;
    }

    public String getProfileCode() {
        return profileCode;
    }

    public void setProfileCode(String profileCode) {
        this.profileCode = profileCode;
    }

    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getCustomerEmail() {
        return customerEmail;
    }

    public void setCustomerEmail(String customerEmail) {
        this.customerEmail = customerEmail;
    }

    public String getCallbackUrl() {
        return callbackUrl;
    }

    public void setCallbackUrl(String callbackUrl) {
        this.callbackUrl = callbackUrl;
    }

    public String getCcToken() {
        return ccToken;
    }

    public void setCcToken(String ccToken) {
        this.ccToken = ccToken;
    }

    public String getSignature() {
        return signature;
    }

    public void setSignature(String signature) {
        this.signature = signature;
    }

}

File: features/payment/payment_setting/src/main/java/com/tokopedia/payment/setting/add/model/CcIframe.kt
package com.tokopedia.payment.setting.add.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CcIframe {

    @SerializedName("merchant_code")
    @Expose
    var merchantCode: String? = null
    @SerializedName("profile_code")
    @Expose
    var profileCode: String? = null
    @SerializedName("ip_address")
    @Expose
    var ipAddress: String? = null
    @SerializedName("date")
    @Expose
    var date: String? = null
    @SerializedName("user_id")
    @Expose
    var userId: String? = null
    @SerializedName("customer_name")
    @Expose
    var customerName: String? = null
    @SerializedName("customer_email")
    @Expose
    var customerEmail: String? = null
    @SerializedName("callback_url")
    @Expose
    var callbackUrl: String? = null
    @SerializedName("cc_token")
    @Expose
    var ccToken: String? = null
    @SerializedName("signature")
    @Expose
    var signature: String? = null

}
